doc/HOWTO_forks.md

```diff
@@ -41,3 +41,19 @@ the commits in your branch so you only need to bring one commit.
 git fetch nu
 git cherry-pick XXXXX
 ```
+
+# Deploy
+From the root of your fork:
+
+- `bundle exec cap [staging_name] deploy`
+
+Find `staging_name` by the name of the `.rb` files in `config/deploy`
```

Comment:
Yeah these are kinda bad notes---why this is still marked WIP haha.

I actually kinda forget what this means, as I recall some of the forks use "stage" and some use "staging". I think this is referring to what that file sets the rails_env as.